org.civicrm.bootstrapcivicrm/css/select2-bootstrap.css

```diff
@@ -0,0 +1,94 @@
+#bootstrap-theme .form-control .select2-choice {
```

Comment:
The _css/_ folder should only contains two post-processed files: *bootstrap.css* and *custom-civicrm.css*

If you want this file to be always part of the bootstrap theme, then i think you can just place it in the `vendor/` folder and add it to the _bootstrap.scss_ file (make sure to remove the `#bootstrap-theme` namespace.